docs/guides/pod-readiness-gates.md

```diff
@@ -0,0 +1,144 @@
+# Pod readiness gate
```

Comment:
Good amount of structure/content sourced from [aws-load-balancer-controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/docs/deploy/pod_readiness_gate.md)